src/megameklab/com/printing/InventoryWriter.java

```diff
@@ -144,6 +146,8 @@ public InventoryWriter(PrintEntity sheet) {
         this.standardBays = new ArrayList<>();
         this.bayFootnotes = new ArrayList<>();
         this.ammo = new TreeMap<>();
+        this.transportBays = sheet.getEntity().getTransports().stream().filter(t -> t instanceof Bay)
```

Comment:
A little difficult to decode. "All transports that are bays and are not living quarters"?

Reply:
Also, is there any point at which this variable becomes stale within this context? I imagine not, since it looks like it's a print action, but...

Reply:
1. Yes
2. No. This instance only exists for a single print action.